integration/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # This Dockerfile builds a base image for the certificate-transparency-go CloudBuild integration testing.
 # See https://hub.docker.com/_/golang for the set of golang base images.
-FROM golang:buster as ct_testbase
+FROM golang:buster@sha256:fd9306e1c664bd49a11d4a4a04e41303430e069e437d137876e9290a555e06fb as ct_testbase
```

Comment:
I think you might need to update the `apt-get` line below - this buster release seems to have renamed the `netcat` package:

```
Step #1: Package netcat is a virtual package provided by:
Step #1:   netcat-openbsd 1.219-1
Step #1:   netcat-traditional 1.10-47
Step #1: 
Step #1: �[91mE: Package 'netcat' has no installation candidate
```

Reply:
I've chosen `netcat-openbsd` instead of `netcat-traditional` due to the following statement.
```
This is the "classic" netcat, written by *Hobbit*. It lacks many features found in netcat-openbsd.
```

https://packages.debian.org/buster/netcat-openbsd
https://packages.debian.org/buster/netcat-traditional